template-for-repository/proofs/Makefile.common

```diff
@@ -365,31 +421,105 @@ arpa: $(ARPA_BLDDIR)
 # Targets to run the analysis commands
 
 $(LOGDIR)/result.txt: $(HARNESS_GOTO).goto
-	$(call DO_CBMC,$(CBMCFLAGS) $(CHECKFLAGS) --unwinding-assertions --trace,$<,$@)
+	$(LITANI) add-job \
+	  --command \
+	    '$(CBMC) $(CBMC_VERBOSITY) --flush $(CBMCFLAGS) $(CHECKFLAGS) --unwinding-assertions --trace $<' \
+	  --inputs $^ \
+	  --outputs $@ \
+	  --pipeline-name $(HARNESS_ENTRY) \
+	  --ci-stage test \
+	  --interleave-stdout-stderr \
+	  --stdout-file $@ \
+	  --ignore-returns 10 \
+	  --tags "stats-group:safety checks" \
+	  --description "$(HARNESS_ENTRY): checking safety properties"
```

Comment:
Should we alphabatize the flags?

Reply:
If you like. I originally had them Christmas-tree sorted. It's up to you, I don't mind that much.
```
	$(LITANI) add-job \
	  --inputs $^ \
	  --outputs $@ \
	  --ci-stage test \
	  --stdout-file $@ \
	  --ignore-returns 10 \
	  --interleave-stdout-stderr \
	  --pipeline-name $(HARNESS_ENTRY) \
	  --tags "stats-group:safety checks" \
	  --description "$(HARNESS_ENTRY): checking safety properties" \
	  --command \
	    '$(CBMC) $(CBMC_VERBOSITY) --flush $(CBMCFLAGS) $(CHECKFLAGS) --unwinding-assertions --trace $<'
```
```python
#!/usr/bin/env python3

import sys

lines = sys.stdin.read().splitlines()
lines = sorted(lines)
lines = sorted(lines, key=lambda x: len(x))
print("\n".join(lines))
```

Reply:
TBH now that I've written a few of these, I'm starting to prefer not alphabetizing them. There's a meaningful order to the flags. `inputs` and `outputs` should go together. `pipeline-name` and `ci-stage` are mandatory, so it's nice for them to go at the top next to `inputs` and outputs`. This means that all of the non-mandatory flags are together at the bottom of the list, so you can easily scan through them and see what's special about each particular job. I wouldn't want something like `ignore-returns` to be close to the top just before `inputs`, because it's unusual and it needs to be noticed.